source/ADIOS.cpp

```diff
@@ -25,10 +25,8 @@
 #include "engine/bp/BPFileReader.h"
 #include "engine/bp/BPFileWriter.h"
 
-#ifdef ADIOS_HAVE_DATAMAN // external dependencies
 #include "engine/dataman/DataManReader.h"
```

Comment:
I am completely fine if we keep the ADIOS_HAVE_DATAMAN and keep DataMan an optional thing. That's actually what I want too, because it will make me less nervous. But I have an impression that I once made something similar work -- loading a shared library in code built as a static library. I don't know if that static library is enough "fully static" though.

Anyway, I will keep DataMan as an optional thing.